core/templates/components/forms/schema-based-editors/schema-based-float-editor.directive.ts

```diff
@@ -112,6 +112,13 @@ angular.module('oppia').directive('schemaBasedFloatEditor', [
             }
           };
 
+          ctrl.NumericInputCustArgEnabled = function() {
```

Comment:
1. Variables and functions start with a small letter.
2. Please do not make interaction specific changes in `schema-based-*-editor` directives. They can be used anywhere and are not specific to interactions.

Reply:
removed